extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRuntimeConfig.java

```diff
@@ -0,0 +1,212 @@
+package io.quarkus.oidc.runtime;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class OidcRuntimeConfig {
```

Comment:
@pedroigor I've just noticed that we also have `OidcConfig` in the runtime package. I'd say we should have this class renamed to `OidcTenantConfig`, while `TenantConfig` to something else, say `CurrentTenantConfig` or `TenantConfigContext`, something like that, what do you think ?

Reply:
Done